File: ShadowFrameworkLite20/src/shadow/image/bitmaps/data/SFPerlineNoiseData.java
package shadow.image.bitmaps.data;

import shadow.system.data.objects.SFBinaryFloatArrayObject;

public abstract class SFPerlineNoiseData extends SFBitmapData{

	protected SFBinaryFloatArrayObject weights=new SFBinaryFloatArrayObject(1,250);

	public SFBinaryFloatArrayObject getWeights(){
		return weights;
	}

	public void setWeights(float... weights){
		this.weights.setValues(weights);
	}

}

File: trunk/ShadowFrameworkJS/src/shadow/image/bitmaps/data/SFPerlineNoiseData.js


function SFPerlineNoiseData(){
	
}

inherit(SFPerlineNoiseData,SFBitmapData);

SFPerlineNoiseData.prototype["prepare"]=function(){
	this.width=new SFShort(0);
	this.height=new SFShort(0);
	this.weights=new SFBinaryFloatArrayObject(1,250);
};